pkg/controllers/disruption/pdblimits.go

```diff
@@ -85,7 +94,7 @@ func (s *PDBLimits) CanEvictPods(pods []*v1.Pod) (client.ObjectKey, bool) {
 }
 
 type pdbItem struct {
-	name                        client.ObjectKey
+	key                         client.ObjectKey
```

Comment:
nit: I think it makes sense to name it the type that it is (or its inferred type of NamespacedName) for ease of understanding
```suggestion
	objectKey                         client.ObjectKey
```

Reply:
Sure, I could go either way on this. We call it `key` elsewhere in the code too rather than `objectKey`